### DISCUSSION TITLE
Life Itself Online Library Access Requests Thread

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#617] [General]
Thread for requesting access to Life Itself online library - see https://lifeitself.org/tao/library

## Steps

Please post a comment here with your zotero username.

**Comment by BoazBF** [comment]
Hey @rufuspollock Please can you add me to the LI online library with this email? bf@boazfeldman.com
That way I have access to zotero.

Many thx and blessings.

**Reply by rufuspollock** [reply]
@BoazBF invite sent. sorry for delay.

**Comment by Alex-Goodall** [comment]
Hi Rufus

Please provide me with access to Life Itself online library. My Zotero username is relevant_education

Thanks!

**Reply by rufuspollock** [reply]
@Alex-Goodall done. sorry for delay - august became a mostly-offline kind of month.

**Comment by Alex-Goodall** [comment]
Not a problem.

Thanks, Rufus,
